### PR TITLE
Task 234 - Hardcode questions content for input variables in OF

### PR DESCRIPTION
**Pull request by will0684**
Brought the questions json from QPS, added OF variables to openfiscaids property.

Brought in and slightly modified tests.

Corrected comments for go-lint.

### CODE REVIEW

File: bindings/benefit_questions.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package bindings
+
+import "github.com/DTS-STN/question-priority-service/models"
```
Comment (evedube) [inline bindings/benefit_questions.go]: I assume this should point to the new service

File: bindings/benefit_questions.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+// BenefitQuestionsRequest is the request sent by the client that contains the benefits
+// for which they would like the list of required questions.
+type BenefitQuestionsRequest struct {
+	// Date period for request in ms since epoch
+	RequestDate int64 `json:"request_date"`
```
Comment (evedube) [inline bindings/benefit_questions.go:9]: Make sure to run go fmt before pushing up your code, if youre using an IDE look for a way to add it on save. Golang has the feature in File Watchers, VSCode should have that option under the Go extension

File: models/question.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package models
+
+type Question struct {
+	ID           string   `json:"id"`
+	Answer       string   `jsons:"answer"`
```
Comment (evedube) [inline models/question.go]: Typo in "jsons"

File: src/questions/load.go [hunk 4]
```diff
@@ -0,0 +1,60 @@
+package questions
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/DTS-STN/question-priority-service/models"
```
Comment (evedube) [inline src/questions/load.go]: Pointing to the old code

**Review by evedube — CHANGES_REQUESTED** [review]

File: src/questions/load.go (hunk 4 above)
Comment (will0684) [reply src/questions/load.go]: corrected references to benefit-service

File: models/question.go (hunk 3 above)
Comment (will0684) [reply models/question.go]: Fixed

File: bindings/benefit_questions.go (hunk 2 above)
Comment (will0684) [reply bindings/benefit_questions.go:9]: Configured to run go fmt

File: bindings/benefit_questions.go (hunk 1 above)
Comment (will0684) [reply bindings/benefit_questions.go]: Fixed

**Review by evedube — APPROVED** [review]

**Pull request merged**